lib/order.rb

```diff
@@ -0,0 +1,81 @@
+require_relative "customer.rb"
+
+class Order
+  attr_accessor :products, :customer, :fulfillment_status
```

Comment:
Similar to the above, these don't need to be `attr_accessor`.  Readers would be fine.